war-in/suu-nft-project
# Create ranks contracts

How to test the interaction:
- install metamask browser extension
- `cd backend && npx ganache-cli --mnemonic "lighten crowd lavish spectacular bustling saw tasteless nauseating panoramic mixed absorbing past"`
- in a new terminal: `truffle console`, then `migrate`
- in a new terminal: `cd frontend && npm start`
- from ganache copy the first private key and import it to metamask
- connect a wallet on the home screen
- create ranks
- go to create events, the ranks should be listed in dropdown
